package-lock.json

```diff
@@ -1141,14 +1141,6 @@
         "@jridgewell/sourcemap-codec": "^1.4.10"
       }
     },
-    "node_modules/@sindresorhus/is": {
```

Comment:
there are massive diffs in the package-lock.json, where could that be coming from?